articles/aks/cluster-configuration.md

```diff
@@ -237,23 +237,7 @@ By contrast, ephemeral OS disks are stored only on the host machine, just like a
 
 Like the temporary disk, an ephemeral OS disk is included in the price of the virtual machine, so you incur no additional storage costs.
 
-Register the `EnableEphemeralOSDiskPreview` feature:
-
```

Comment:
Did you intended to the remove the feature flag? That makes this a GA feature 😊

Reply:
whoops

Reply:
let's try that again